### PR TITLE
Added collaboration info

### PR DESCRIPTION
**Pull request by matteo1976**
Added on README file the info for start to collaborate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -363,6 +363,30 @@ const options = {
 }
 ```
 
+## Collaboration
```
Comment (gabrielliwerant) [inline README.md]: How about calling this `Contributing` instead, as that's the more common title for this kind of section.

File: README.md [hunk 2]
```diff
@@ -363,6 +363,30 @@ const options = {
...
+Thanks for taking an interest in the library and the github community!
+
+The following commands should get you started:
+
+Fork the project and clone the project in a local repository
+Add a new brunch
```
Comment (gabrielliwerant) [inline README.md]: We can probably remove this line and the extra line break below. I think we can leave out specific branch and naming conventions for now, as forking is good enough to get started.

File: README.md [hunk 3]
```diff
@@ -363,6 +363,30 @@ const options = {
...
+npm run dev
+```
+
+open  http://localhost:5050/ in browser
+
+After you make your changes locally, you can run the test suite with `npm test`. 
+
+Adding additional tests around existing functionality or any of your proposed changes is always appreciated.
+
+Make a Pull request
```
Comment (gabrielliwerant) [inline README.md]: I think we can remove this line and the "Good Work." There isn't any other way to submit changes except for opening a pull request, so I think short and sweet is best for the instructions for now.

File: README.md [hunk 4]
```diff
@@ -363,6 +363,30 @@ const options = {
...
+Fork the project and clone the project in a local repository
+Add a new brunch
+
+
+```sh
+npm i
+npm run dev
+```
+
+open  http://localhost:5050/ in browser
```
Comment (matteo1976) [inline README.md]: I think this line is important

File: README.md [hunk 5]
```diff
@@ -363,6 +363,30 @@ const options = {
...
+
+After you make your changes locally, you can run the test suite with `npm test`.
```
Comment (matteo1976) [inline README.md]: I think it's important to know how make test

**Comment by gabrielliwerant** [comment]
This looks good to me! It should help people new to the project to get started, and we can add additional instructions as needed.

**Pull request merged**